lib/srv/db/server.go

```diff
@@ -358,6 +435,13 @@ func (s *Server) startDatabase(ctx context.Context, database types.Database) err
 		// Log, but do not fail. We will fetch the version later.
 		s.log.Warnf("Failed to fetch the MySQL version for %s: %v", database.GetName(), err)
 	}
+
+	if database.GetProtocol() == defaults.ProtocolSQLServer {
+		if err := s.startSQLServerLDAP(ctx, database); err != nil {
+			return trace.Wrap(err)
+		}
```

Comment:
Why do we want always to start SQLServerLDAP ? For the old configuration when the `database.GetAD().LDAPCert` is empty the `startSQLServerLDAP` will returns an error and block the service from running.